### DISCUSSION TITLE
Github Release results in 500 server error

### DISCUSSION THREAD

**Discussion by dibbla** [community/community#55680] [Repositories] [answered]
### Select Topic Area

Question

### Body

I was working on my repo [Here](https://github.com/dibbla/PaperDino-Everyday_arXiv), adn I want to publish a new release for it. However, it keeps resulting in 500 error, and the new release is always a draft.

I'm using tag `v1.1` and a .zip file uploaded.

**Comment by greendoescode** [comment, marked as answer]
![image](https://github.com/community/community/assets/77177015/b5faf43b-66a9-47bd-9217-4f7c0356106f)
Servers down rn

**Comment by tjlauer** [comment]
Major issues. See https://www.githubstatus.com/

**Comment by dibbla** [comment]
Thanks @tjlauer !
